src/styles/blocks/calendar-loops.scss

```
@@ -0,0 +1,23 @@
+@each $day, $margin in $days {
+  .calendar--#{$day} > .calendar__day:first-child {
+    margin-left: ($block-size + 1) * $margin;
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar > .calendar__day:nth-child(#{$i})::before {
+    content: "#{$i}";
+  }
+}
+
+@for $i from 28 through 31 {
+  .calendar--#{$i} > .calendar__day:nth-child(n + #{$i}) {
+    display: none;
+  }
+}
+
+@for $i from 28 through 31 {
+  .calendar--#{$i} > .calendar__day:nth-child(-n + #{$i}) {
+    display: flex;
+  }
+}

```

I have no idea how you come up with this idea, no pun intended. Try this instead

```suggestion
@for $i from 28 through 31 {
  .calendar--#{$i} :nth-child(n + #{$i + 1}) {
    display: none;
  }
}
```